cosmos_rl/policy/model/base.py

```diff
@@ -335,11 +335,12 @@ def policy_maybe_decompose_weights_to_hf_naming(self, name, param):
     @abstractmethod
     def rollout_prepare_recv(
         self, vllm_model: Any
-    ) -> Tuple[Dict[str, torch.Tensor], List[List[Tuple[str, int]]]]:
+    ) -> Tuple[Dict[str, torch.Tensor], List[Tuple[str, int]], List[List[str]]]:
```

Comment:
this will break existing out of source tree weight mapper

Reply:
yeah, here we do this change to separate the ordered param names and group information to simplify the calculation in controller.

Reply:
Have changed back this interface.